include/nbl/asset/utils/CJITIncludeLoader.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+#define _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+
+#include "nbl/asset/utils/IShaderCompiler.h"
+#include "nbl/video/SPhysicalDeviceFeatures.h"
+#include "nbl/video/SPhysicalDeviceLimits.h"
+
+#include <string>
+
+using namespace nbl::asset;
```

Comment:
https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers

you'lll just have to write `asset::` in the header